stock_picking_move_package_to_package/views/stock_package_level_view.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record
+        id="package_level_tree_view_picking_move_package_to_package"
+        model="ir.ui.view"
+    >
+        <field name="name">Package Level Tree Picking Move Package To Package</field>
+        <field name="model">stock.package_level</field>
+        <field name="inherit_id" ref="stock.package_level_tree_view_picking" />
+        <field name="arch" type="xml">
+            <field name="package_id" position="after">
+                <field
+                    name="package_dest_id"
+                    attrs="{'readonly': [('state', 'in', ('confirmed', 'assigned', 'done', 'cancel'))]}"
```

Comment:
@angelinaanaki Could you explain a little bit your use case as it is a little bit too restrictive.

I can't set a destination package in a normal flow (tested on runboat).

![image](https://github.com/OCA/stock-logistics-workflow/assets/19529533/3405e965-4a25-4af2-8094-6b37f1101df3)

Reply:
@rousseldenis It is important that the user can add the destination package in all the states where he can select a source one. In this case, the source package is also read-only.

Reply:
@angelinaanaki Ok, so, USAGE and DESCRIPTION fragments should be a little bit more explicits. Thanks